vrouter.go

```diff
@@ -397,6 +480,78 @@ func (self *Vrouter) UpdateLocalEndpoint(endpoint *OfnetEndpoint, epInfo Endpoin
 	return nil
 }
 
+// AddHostPort sets up host access.
+func (self *Vrouter) AddHostPort(hp HostPortInfo) error {
+	if hp.Kind == "NAT" && self.hostNATInfo.PortNo != 0 {
+		log.Errorf("Host NAT port exists: %+v", self.hostNATInfo)
+		return fmt.Errorf("Host NAT port exists")
+	}
+
+	ipDa, daMask, err := ParseIPAddrMaskString(hp.IpAddr)
+	if err != nil {
+		log.Errorf("Bad host route - %v", err)
+		return err
+	}
+
+	netMask := pflag.ParseIPv4Mask(daMask.String())
+	maskedIP := ipDa.Mask(netMask)
+
+	// Save the info
+	self.hostNATInfo = hp
+
+	// Set up DNAT for ingress traffic
+	inNATFlow, _ := self.inputTable.NewFlow(ofctrl.FlowMatch{
+		InputPort: self.hostNATInfo.PortNo,
+		Priority:  FLOW_MATCH_PRIORITY,
+	})
+	inNATFlow.Next(self.hostDNATTable)
+
+	// Packets from hostport that miss in IP table are dropped
+	inNATMiss, _ := self.ipTable.NewFlow(ofctrl.FlowMatch{
+		InputPort: self.hostNATInfo.PortNo,
+		Priority:  FLOW_FLOOD_PRIORITY,
```

Comment:
No, we need this to be higher than MISS because this is a specific rule applicable only to NAT port.